src/components/molecules/Banner.tsx

```diff
@@ -25,11 +32,22 @@ const Banner = () => {
         </h2>
       </div>
       <div className="center lg:flex-row flex-col  gap-4 z-10">
-        <BaseButton
-          onClick={redirectToRegister}
-          className="bg-gray w-[150px]"
-          text="Unirme"
-        />
+        <>
+          {isLogged ? (
+            <BaseButton
+              onClick={() => redirectToProfile(user.uid)}
```

Comment:
please move this into its own function

Reply:
Since we already have user's info, we don't need to pass this uid as an argument, just change redirectToProfile function to include this id already